.gitignore

```diff
@@ -121,6 +121,5 @@ junit.xml
 # ms IDE stuff
 *.code-workspace
 .history
-.vscode
```

Comment:
How will this removal affect VS Code users that might have a custom `settings.json`? For example to store `python.pythonPath` pointing to their virtual environment.

Looks like they might commit this file by mistake if it is not ignored anymore?

Reply:
Added `.vscode/settings.json` as a more specific exclusion.